wagtailmenus/templatetags/menu_tags.py

```diff
@@ -347,15 +358,20 @@ def section_menu(
         current_level=1,
         max_levels=max_levels,
         apply_active_classes=apply_active_classes,
-        allow_repeating_parents=allow_repeating_parents
+        allow_repeating_parents=allow_repeating_parents,
+        use_absolute_urls=use_absolute_urls,
     )
 
     """
     We want `section_root` to have the same attributes as primed menu
     items, so it can be used in the same way in a template if required.
     """
     setattr(section_root, 'text', section_root.title)
-    setattr(section_root, 'href', section_root.relative_url(site))
+    if use_absolute_urls:
+        url = section_root.full_url
```

Comment:
Yeah it looks like it would be better for performance, since they use the request to cache the root path lookup.